src/home/components/Connect.jsx

```diff
@@ -0,0 +1,232 @@
+State.init({
+  startIndex: 0,
+  endIndex: 2,
+});
+
+const DescriptionHeader = styled.h2`
+  color: #f4f4f4;
+  font-size: 28px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 120%; /* 43.2px */
+`;
+
+const Description = styled.p`
+  color: #f4f4f4;
+  text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 120%; /* 28.8px */
+`;
+
+const imageSource =
+  "https://s3-alpha-sig.figma.com/img/3b54/e4fa/1508a7cf222faf5f331f489d224e4fd6?Expires=1698624000&Signature=gIhYLppLobWi6QMrBzxH-SQFd5JGKJrah~PHlxUYgOOoYPxOTTQ9qOEB8xqHX8pGuaJqTre7Fn~FdI~jWK0WdPrvev8OJDIKWF1zSLSLYG20ztuMR8K8hu4~N1k-Fomyp34H6GalsbWM6p~J5QS0G5L-Wkc2am24-Ys8c4BwqH-CdPAvehTNxgwgGq77aLkf1HqfRaUW5uLcLBhzOavk03lXSxSTmUo5KZX5SOxYTpktmLozASkkFR4-SAVQkxEwlN8s3JG8eX7dsUohsfiE-jYFjuKxfwU~f0OfyvNCvdyvEg335qIMRvD5NCrmQTnm0ZJRxOX~imYi6GV74U8p8w__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4";
```

Comment:
https://ipfs.near.social/ipfs/bafkreidwtqu7wrppkvwb2criwgk6u7bfx7ojyaempio7pnkhvspxxeujke

Btw, I'm using this:
https://near.social/cuhoangng.near/widget/ImageUploader

Reply:
Thanks for telling me